### PR TITLE
humanoidmodelwalk and humanoidstand

### PR DESCRIPTION
**Pull request by zdx3578**
add env humanoidmodelwalk and humanoidstand.

### CODE REVIEW

File: gym/envs/mujoco/humanoidmodelwalk.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                               data.cvel.flat,
+                               data.qfrc_actuator.flat,
+                               data.cfrc_ext.flat])
+
+    def _step(self, a):
+        pos_before = mass_center(self.model)
+        foot1=self.model.data.xipos[6][0]
+        foot2=self.model.data.xipos[9][0]
+        self.do_simulation(a, self.frame_skip)
+        global i
```
Comment (tlbtlbtlb) [inline gym/envs/mujoco/humanoidmodelwalk.py]: Please don't do this

File: gym/envs/mujoco/humanoidmodelwalk.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+        pos_after = mass_center(self.model)
+        alive_bonus = 5.0
+        data = self.model.data
+        lin_vel_cost = 0.25 * (pos_after - pos_before) / self.model.opt.timestep
+        quad_ctrl_cost =  0.1 * np.square(data.ctrl).sum()
+        quad_impact_cost = .5e-6 * np.square(data.cfrc_ext).sum()
+        quad_impact_cost = min(quad_impact_cost, 10)
+        modelreward=  1 / ( 20 + (self.model.data.xipos[6][1]+ self.model.data.xipos[9][1]))
+        walkreward=  (self.model.data.xipos[6][0]-foot1) if (i % 2) else  (self.model.data.xipos[9][0]-foot2)
```
Comment (tlbtlbtlb) [inline gym/envs/mujoco/humanoidmodelwalk.py]: Alternating the source of rewards every timestep is strange. Why not average these two values?

File: gym/envs/mujoco/humanoidstand.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        pos_before = mass_center(self.model)
+        self.do_simulation(a, self.frame_skip)
+        pos_after = mass_center(self.model)
+        alive_bonus = 15.0
+        data = self.model.data
+        lin_vel_cost = 0.25 * (pos_after - pos_before) / self.model.opt.timestep
+        quad_ctrl_cost =  0.1 * np.square(data.ctrl).sum()
+        quad_impact_cost = .5e-6 * np.square(data.cfrc_ext).sum()
+        quad_impact_cost = min(quad_impact_cost, 10)
+        reward = alive_bonus - lin_vel_cost - quad_ctrl_cost - quad_impact_cost
```
Comment (tlbtlbtlb) [inline gym/envs/mujoco/humanoidstand.py]: Subtracing lin_vel_cost will give rewards when it walks backwards, which probably isn't what you want.

File: gym/envs/mujoco/humanoidmodelwalk.py (hunk 2 above)
Comment (zdx3578) [inline gym/envs/mujoco/humanoidmodelwalk.py]: think to make it every time make large step

File: gym/envs/mujoco/humanoidstand.py (hunk 3 above)
Comment (zdx3578) [inline gym/envs/mujoco/humanoidstand.py]: yes, train result , i had see this .

Comment (zdx3578) [inline gym/envs/mujoco/humanoidstand.py]: i fix two

**Pull request closed without merging**